notebooks/tutorial/1 - Setup.ipynb

```diff
@@ -63,13 +63,13 @@
     "import os\n",
     "import platform\n",
     "\n",
-    "header_name = 'header_tome'\n",
+    "header_name = 'header_tome.2022-05-15,2022-05-15'\n",
     "ds_type = 'csds'\n",
     "\n",
     "if collection_path == 'PATH/TO/TOMES':\n",
-    "    collection_path = os.path.join('..','..','tmp','tomes')\n",
```

Comment:
I think we can make these the same by default. What if we write it like


1. Where you will save the data science files.
2. Where you will save your tomes (views of the data).
3. What to call your header tome.

```python
tome_collection_path = '../../tmp'
ds_collection_path = '../../tmp'
header_name = 'header_tome.2022-05-15,2022-05-15'
```

```python
ds_type = 'csds'
tome_collection_path = os.path.abspath(tome_collection_path)
ds_collection_path = os.path.abspath(ds_collection_path)
```

Reply:
kk